src/styles/styles.js

```diff
@@ -766,6 +766,13 @@ const styles = {
         opacity: 0.6,
     },
 
+    chatItemMessageLink: {
+        color: colors.blue,
+        fontSize: variables.fontSizeNormal,
+        fontFamily: fontFamily.GTA,
+        lineHeight: 20,
```

Comment:
It would be great to turn this into a variable. The 20 value is referenced many times and we probably want to make sure those are all linked to the same variable.

Reply:
I agree with the improvement but would prefer not to block this PR over this change if possible -- simply because each small change delays N5 and this pattern is not introduced with this PR.

Please let me know if you disagree.

Reply:
I agree :D